apps/dashboard/app/apps/nav_config.rb

```diff
@@ -1,11 +1,18 @@
+# frozen_string_literal: true
+
 #
 # Deprecated.
 # This is now configurable through configuration. See UserConfiguration class.
 class NavConfig
-    class << self
-      attr_accessor :categories, :categories_whitelist
-      alias_method :categories_whitelist?, :categories_whitelist
-    end
-    self.categories = ["Apps", "Files", "Jobs", "Clusters", "Interactive Apps"]
-    self.categories_whitelist = false
+  class << self
+    attr_accessor :categories, :categories_allowlist
+    alias categories_allowlist? :categories_allowlist
```

Comment:
Is there a reason the alias is a boolean call but the symbol is not?

Reply:
It's a rubocop style, though I don't know why the 2nd arguement could be a symbol and not get flagged?  I've updated to use barename for both arguments. 

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/Alias